### PR TITLE
:bath: cleaned up some code

### PR DESCRIPTION
**Pull request by ajthomascouk**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-120.dev.beta.nhschoices.net)

File: app/views/error-404.nunjucks [hunk 1]
```diff
@@ -5,7 +5,11 @@
 {% block pageTitle %} {{ message }} {% endblock %}
 
 {% block header %}
-  <h1 class="local-header--title--question">{{ message }}</h1>
+<div>
+  <div class="local-header--question">
+    <h1 class="local-header--title--question">{{ message }}</h1>
```
Comment (beseven) [inline app/views/error-404.nunjucks]: Based on BEM, wouldn't this be `--question--title`?

Comment (ajthomascouk) [inline app/views/error-404.nunjucks]: This is legacy code, but yeah good shout.

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-120.dev.beta.nhschoices.net)

File: app/views/error-404.nunjucks (hunk 1 above)
Comment (beseven) [inline app/views/error-404.nunjucks]: Raised as issue here https://github.com/nhsuk/gp-finder/issues/121

**Pull request merged**